common/views/components/IIIFBookReader/IIIFBookReader.js

```diff
@@ -21,7 +22,14 @@ const IIIFBookReader = ({
             .canvases
             .slice(0, 15)
             .map(canvas => {
-              return (<div key={canvas.thumbnail['@id']} ><img src={canvas.images[0].resource['@id']} /></div>);
+              return (<div key={canvas.thumbnail['@id']} style={{position: 'relative', 'marginBottom': 30}}>
+                <ImageViewer
```

Comment:
Are we loading in loads of DOM here?
Just asking as we are actually starting to get quite heavy.

Reply:
Not much no. Might be worth thinking about how we measure performance of this though